packages/cozy-scanner/Readme.md

```diff
@@ -0,0 +1,32 @@
+# Cozy Scanner
+
+Cozy-Scanner provides a component to take a picture of a document and describe it
+
+## Installation
+
+Cozy-Scanner requires :
+
+- a cordova plugin : `cordova plugin add https://github.com/cozy/cordova-plugin-camera.git#bf519ca009eeaec3a47237ee07a09d515cc733df`
+- At least Cozy-UI 25.7
+- Cozy-Doctypes
+
+To use the component and get the style, you need to import the css in your app
+
+```js
+import "cozy-scanner/dist/stylesheet.css";
```

Comment:
It would be nice not to have to do that ? Why is this necessary ? Could the scanner component add this stylesheet by itself ?

Reply:
At the beginning I wanted to let the app dealing with the style without having to import a css file. But `scanner` use a few stylus files, and this means the app needs to be able to handle stylus file and this is something I want to avoid for the sake of simplicity to use. 

So the option was to generate a stylesheet.css file. At this moment, we have two choice : 
1/ Having each of our component importing this stylesheet 
2/ Let the app importing the stylesheet 

Again, I choose the 2 because copy / pasting a line, is simpler for DX. Having the same stylesheet imported in each component will result in a `big bundle size` (right?) or... you'll need to customize your build tools and this is something I try to avoid.

Reply:
> Having the same stylesheet imported in each component will result in a big bundle size (right?) 

No. The stylesheet will be only once in the bundle.

Reply:
Removed the need of stylus since the CSS is pretty easy and having to import manually the stylesheet in the transpiled code was not very easy